base_layer/p2p/src/auto_update/gpg_keys/hansieodendaal.asc

```diff
@@ -0,0 +1,13 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+mDMEXFAOHRYJKwYBBAHaRw8BAQdA5Pr1oWCXuk4KOLww8nzozn1R/TvL5AvIdScU
+9R7W2D+0JEhhbnNpZSBPZGVuZGFhbCBFQ0MgPHBsdXRvQHRhcmkuY29tPoiWBBMW
+CAA+FiEEiTBA/hklxU/poczD00Haf8YJhicFAlxQDh0CGyMFCQlnaQMFCwkIBwIG
+FQoJCAsCBBYCAwECHgECF4AACgkQ00Haf8YJhid3AQD/elT+/dn+IdfEGf4Veu4L
+VdE0uAmg+3JKcxSFAzuJD1sA/23QkxDAqDnBVxX1vqLzMx4WmB30w8Qqjd6NSuqF
+WD0AuDgEXFAOHRIKKwYBBAGXVQEFAQEHQJDBWycKABkLFhl+/3wq9jXFQNKu2fhl
+bG8o6qkFgdIHAwEIB4h+BBgWCAAmFiEEiTBA/hklxU/poczD00Haf8YJhicFAlxQ
+Dh0CGwwFCQlnaQMACgkQ00Haf8YJhifHDQEA8wC6rEJ0q3hzi7gPhwv/J1waspjd
+ic5qhFtth9VSH64A/3nwUCTxOfSN4O2BJbEyIbTvNKrqfU0xfIPb5fjLz40H
+=mR5Q
```

Comment:
```suggestion
mDMEXFAOHRYJKwYBBAHaRw8BAQdA5Pr1oWCXuk4KOLww8nzozn1R/TvL5AvIdScU
9R7W2D+0JEhhbnNpZSBPZGVuZGFhbCBFQ0MgPHBsdXRvQHRhcmkuY29tPoiWBBMW
CAA+AhsjBQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAFiEEiTBA/hklxU/poczD00Ha
f8YJhicFAmW4lxAFCR5fHEEACgkQ00Haf8YJhicyVQD/RrWwkz5z8e5HgrinBVFH
hfAoCmPBQkI5+zBndtnnYo8A/jcj/AIH4h3lsxPuC2DG3mMoAsr2/MVxP1f+WHtg
XvkCiQIzBBABCAAdFiEEriHzWWFX1DYW7Df67QUeLkD7PZ0FAlxaoR4ACgkQ7QUe
LkD7PZ1GVhAAiQR/aoYL13Sj0Xh1CgVYMoYCGm+ulpi/s5U42WxiMw4SOFVqdI7q
0tHCS0IyxomBXQrW7UsNne4ErfmL6D2j2CS5pkBaG+6/gzCFk18zXdRUC87no8rE
+DIjh6XIPNvxxJmxzPJBN1JLiI9xsQGcFGOJE/MSL1PxpXZ/n/zYoxQqvz1eqNy0
nHiuHSTwuTc+/4Quu48U2115ttqcqe8QZJ224oDjQRtCNIzzQUwSmw05OySsq2Zx
8q2PwWBeElKDIVT8sIdl/VSKU41j+Tt8Bvqi3Av/Egu94d+VhdXDMlBctl3ana23
WYyjxm8U0onu5Z2K6CD3SSZoXsTOgY2eYR5pWigT3JoceI5nHhRxXxpN5yCKZ4ap
tfjzEuUjLGtL30b9QsK7cwQK9lC1nmvyvzcu/BkkewAh+fmYnckuhNtTv/XRtgto
TIaESKQ+vOxBa2qxbvDKKSkEbchPLp/zqjcB7APtl3bGei5fMDb3LNJvKlmUCySi
lghBOJHjXCJGr3d/oGWz+Lr9Dk57KRRkUxpspN2hMmISDib1J2Pmwh/ZbBsy7gpU
mISAkq+yIShBsddAxUVZ2JIwd2Lqe+K0dXmvgFvSPTRtRZaPiWWyqKkDGw/fmN7I
oX2ZqgmSLZBPkmcyJHwp2OYpkgwZvOdV9RNEafG/p5RD/LDQKoDOp7+JAjMEEAEI
AB0WIQSaNRgoWBvbb8MsADdrUDZi9rVc4gUCXFqsFwAKCRBrUDZi9rVc4lcQD/4p
YST4cg8ZaurMMqPd4zbWJsj5inW/2MaiWt6fgm0P+OCSy84i3WF2cgWGAq9z6aM+
IMKLNJh+diu6lqbDTvANqQPCOB+GIQHsQMVU00mcpAbgZATTanZjp6DaK/6s7qRs
3sBrcBdOphMZl4wfF09LAAFuWDEVNNY/DRCywMBEkKXBBOpYFPaah40i6cSogegs
ZmclthXqfTfRs1zLBCPGLqEbiJ1nOgAdWM4vsNZw7I4VlTAN5w+1RaaOOMP9Sy83
H5ZgJ5i3Jfjx7qb7nrIRwLZe7d3unkt6FGboRODJkoUc0Q4M9QNTEhTJgJEHub2L
UKemJfPwLFp++MCsXgC9pI1SsAc1+KA5HFMh1PKbNIh6JCZX2BN2ZjAY3GSaczwM
BXLJ4ouDX/iDztEV16RjpEVWCF/s+gUdHTT9dzSuf5svYdJSSdBRH0jMqCXhhQbS
Xh/9ZAh1iqjoJf8dKSpVQphbE0B2iZIyZXJghFoZb/CF/y/idvXCFQ95lVuSXy/v
jBd3RjWGZcBATpCl9ie8OZP4jyCQQt4YzJy1oyWMVu6fWk3v6dK2g86N21Ci2Tkw
EspYV2Q4RMfh0LThulLhNgZYAlMwCRPq1Pyl63eYOwb5U4ClTIiRJNyamDiUMZkP
IAfLZmuX8dZKey3xRTN6XY5KkPFSdkUyql0pgWk7Q4h1BBAWCAAdFiEEXEnXYm0J
2S2WpzH4BPPYqOZYQswFAlxapDkACgkQBPPYqOZYQsy33AD6AzO52ZpS2pz89sV2
A2ZE7Lmh7qbj7OZl+r36KvnS9gUA/A/nOown4pzNfMnh6lLBIpNkLp752D5LVCM7
zgpRGN4GiQIzBBABCAAdFiEE6dgmhm4UIrfqKEWccigxfjhEf0oFAlxiudQACgkQ
cigxfjhEf0rV4g//blrFre5rDJCmlTepzBcZ59Bj3840j1gnbI7IRaQOlIu4l1G0
QFiamzRZcg2jovCPMWIVoDbg8ejH3kB+PPHx635FlLKGqlBL0uvE2lxyj2FBzVH1
+VnHlxFObMdg6CmOixLrEjYV64/bmqOQ4scM+MQfwsJgBIjtrazpndoKDNActP/q
S77UnNA85OdSUa16odz/miLj34ov4I1wBFv4xWEJGvyvtMgN+qjJfUGB5nHW7Ya7
Ythzr0Y244tCbnkRsiNqJCw6N6PgG3HBJqL7W9P5zI7o/JL6XSPPajIdc/qz4B9K
EObiNJyBujGPHiYOgsQESwx7qi8131bdxXCLDYTRJ+8SsyjCcVghSZ4fVJTJzHCK
wJ0J4MqJQvqOxpYg3xbgjkIOUoqtjFQSjL42RTgv8siafsYTFJMaoA0hjbWYnlza
I8noZlryxlzkEpKMDCwqjVVRoR17Gr2cXKCWvk9XN3sUsY7Y2AD5GC5W+w92dOk2
+1duFfwIcSUmuWDeDNDTSPlALbqC1C+L2rxIzqQk0JJ6WAolMA60TmwUOGWus1Ot
3KRWoOjSK9bCKDnRXoRjwYlVkJqm6ksCisUOVbOZVJSSoIuvK9f1ePAbE/UryZDA
rTnmt5vUuNT/y/4vLUc0hACk5oh40x1+AtWd4uCzXA/wOtPVglqif53sXcO4OARc
UA4dEgorBgEEAZdVAQUBAQdAkMFbJwoAGQsWGX7/fCr2NcVA0q7Z+GVsbyjqqQWB
0gcDAQgHiH4EGBYIACYWIQSJMED+GSXFT+mhzMPTQdp/xgmGJwUCXFAOHQIbDAUJ
CWdpAwAKCRDTQdp/xgmGJ8cNAQDzALqsQnSreHOLuA+HC/8nXBqymN2JzmqEW22H
1VIfrgD/efBQJPE59I3g7YElsTIhtO80qup9TTF8g9vl+MvPjQc=
=UPcF
```

Reply:
This should be a different PR. @SWvheerden please exclude this file for now